float64[6] pwm
